src/main/java/de/unistuttgart/overworldbackend/controller/LectureController.java

```diff
@@ -11,16 +20,30 @@
 @RequestMapping("/lectures")
 public class LectureController {
 
+  @Autowired
+  private LectureRepository lectureRepository;
+
+  @Autowired
+  private LectureService lectureService;
+
+  @Autowired
+  private LectureMapper lectureMapper;
+
   @Operation(summary = "Get all lectures")
   @GetMapping("")
```

Comment:
```suggestion
  @GetMapping("/")
```
Isn't itt better to call the mapping `/` cause normally a `/` is prefixed?

Reply:
I think it's a normal REST convention not to use a trailing slash

Reply:
Okay, I'm not familiar with it. It just seemed strange to me.